randovania/games/samus_returns/logic_database/Area 5 - Tower Lobby.txt

```diff
@@ -893,15 +910,11 @@ Extra - asset_id: collision_camera_014
   > Pickup (Phase Drift)
       Any of the following:
           Morph Ball
-          # https://youtu.be/P18XiY9K36o
+          # Corner Clip: https://youtu.be/P18XiY9K36o
           Corner Clip (Intermediate) and Out of Bounds Movement (Intermediate)
```

Comment:
We want to delete the Corner Clip trick entirely, so make this just oob movement (and delete the Trick from the header)

Reply:
will delete from the header at a later date to ensure less conflicts